internal/handler/web/studies.go

```diff
@@ -215,19 +215,12 @@ func (h *Handler) PostStudiesStudyIdAssets(ctx *gin.Context, studyId string) {
 }
 
 func (h *Handler) GetStudiesStudyIdAssetsAssetId(ctx *gin.Context, studyId string, assetId string) {
-	studyUUID, err := parseUUIDOrSetError(ctx, studyId)
-	if err != nil {
-		return
-	}
-
-	assetUUID, err := parseUUIDOrSetError(ctx, assetId)
+	uuids, err := parseUUIDsOrSetError(ctx, studyId, assetId)
```

Comment:
I don't see this function anywhere else in this PR, should it be singular instead of plural?